awslimits/manage.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sendgrid
+from flask_script import Manager
+
+from server import app
+from support import get_limits_for_alert, alert_email_body
+
+manager = Manager(app)
+
+
+@manager.command
+def send_alerts():
```

Comment:
Maybe in the future we should we keep track somehow of things we already notified to avoid spamming people.